mono/mini/debugger-agent.c

```diff
@@ -4279,6 +4437,8 @@ insert_breakpoint (MonoSeqPointInfo *seq_points, MonoDomain *domain, MonoJitInfo
 	DEBUG_PRINTF (1, "[dbg] Inserted breakpoint at %s:[il=0x%x,native=0x%x] [%p](%d).\n", mono_method_full_name (jinfo_get_method (ji), TRUE), (int)it.seq_point.il_offset, (int)it.seq_point.native_offset, inst->ip, count);
 }
 
+#ifndef IL2CPP_MONO_DEBUGGER
+
 static void
 remove_breakpoint (BreakpointInstance *inst)
```

Comment:
from what I remember of what we did for our in editor debugger during hackweek, these add_breakpoint and remove_breakpoint functions are 100% needed. When a user clicks a line in VSTU or other debugger these functions are called and tell mono where to break..not sure how it can work without them